Find Lua 2.1 development header files. Thanks to Damien R. for the patch
--- a/cmake/FindLuaJIT.cmake
+++ b/cmake/FindLuaJIT.cmake
@@ -31,7 +31,7 @@
 FIND_PATH(LUAJIT_INCLUDE_DIR lua.h
   HINTS
   $ENV{LUAJIT_DIR}
-  PATH_SUFFIXES include/luajit-2.0 include/luajit2.0 include/luajit include
+  PATH_SUFFIXES include/luajit-2.0 include/luajit2.0 include/luajit include include/luajit-2.1
   PATHS
   ~/Library/Frameworks
   /Library/Frameworks
